unit-tests/py/rspy/devices.py

```diff
@@ -209,26 +196,24 @@ def query( monitor_changes=True, hub_reset=False, recycle_ports=True ):
     Start a new LRS context, and collect all devices
     :param monitor_changes: If True, devices will update dynamically as they are removed/added
     :param recycle_ports: True to recycle all ports before querying devices; False to leave as-is
-    :param hub_reset: Whether we want to reset the Acroname hub - this might be an better way to
+    :param hub_reset: Whether we want to reset the hub - this might be a better way to
         recycle the ports in certain cases that leave the ports in a bad state
     """
     global rs
     if not rs:
         return
     #
     # Before we can start a context and query devices, we need to enable all the ports
-    # on the acroname, if any:
-    if acroname:
-        if not acroname.hub:
-            acroname.connect( hub_reset )  # MAY THROW!
-
-            if recycle_ports:
-                acroname.disable_ports( sleep_on_change = 5 )
-                acroname.enable_ports( sleep_on_change = MAX_ENUMERATION_TIME )
-
-            if platform.system() == 'Linux':
-                global _acroname_hubs
-                _acroname_hubs = set( acroname.find_all_hubs() )
+    # on the hub, if any:
+    if hub:
+        if not hub.is_connected():
+            hub.connect(hub_reset)
+        hub.disable_ports( sleep_on_change = 5 )
+        hub.enable_ports( sleep_on_change = MAX_ENUMERATION_TIME )
+
+        if platform.system() == 'Linux':
+            global _hubs
+            _hubs = set(hub.find_all_hubs())
```

Comment:
`hub.find_all_hubs()` is confusing: it looks like a method of `hub` but it isn't: it's a function inside `device_hub`. It should be `device_hub.find_all_hubs()`

Reply:
Ohhh I see what you did there... nice.